src/models/privatedatastore.js

```diff
@@ -0,0 +1,38 @@
+export default {
```

Comment:
How is the `privatedatastore` namespace being used? What constitutes a private controller? We should be able to use the `global` namespace to track user state regardless of whether the user is authenticated.

Reply:
Moved selected controllers by the user to `global` namespace. I am using `privateControllers` for keeping track of the mocked controllers of user(will be replaced with `fetchControllers` in `dashboard` namespace later once we have data in server).